qutebrowser/browser/commands.py

```diff
@@ -911,12 +911,22 @@ def toggle_inspector(self):
                 cur.inspector.show()
 
     @cmdutils.register(instance='command-dispatcher', scope='window')
-    def download_page(self):
-        """Download the current page."""
-        page = self._current_widget().page()
+    def download(self, url=None, dest=None):
+        """Download a given URL, or current page if none given.
+
+        Args:
+            url: The URL to download, or None to download current page.
+            dest: The file path to write the download to, or None to ask.
```

Comment:
Here `None` makes sense from the developer's perspective, but not if an user reads it in the command documentation. Probably *not given* or so would make more sense?